src/Unleash/Metrics/ToggleCount.cs

```diff
@@ -27,8 +27,7 @@ public void Register(bool active)
 
         public void Register(string variantName)
         {
-            var current = variants.GetOrAdd(variantName, s => new long());
-            Interlocked.Increment(ref current);
+            variants.AddOrUpdate(variantName, 1, (k, v) => Interlocked.Increment(ref v));
```

Comment:
I think we don't need Interlocked here. afaik a long is not a reference type - so it's not changed "in place" it's just copied and the new value is returned and set in a thread safe manner. I don't think it'll cause harm keeping it in though

Reply:
Good eyes, agreed, it's noise here. It's not harmful to keep, outside of forcing an unnecessary lock, but it does nothing useful

Reply:
Ah, that's a good point! I'm happy to make that change. Should be updated in befbe3e05ee44e3be66a470ba46c760c1c239769